src/dom_cfg/rpi_5.c

```diff
@@ -41,8 +41,40 @@ static ssize_t get_domu_image_size(void *image_info, uint64_t *size)
 }
 #endif /* CONFIG_DOM_STORAGE_FATFS_ENABLE */
 
+#if defined(CONFIG_PV_DOMAIN)
+static const char *params_vif =
+	"vif=[ 'backend=1,bridge=xenbr0,mac=08:00:27:ff:cb:ce,ip=192.168.0.2 "
+	"255.255.255.0 192.168.0.1' ]";
+
+static const char *params_vbd =
+	"disk=[ 'backend=1, format=raw, vdev=xvda, "
+                                "access=rw, target=/etc/u-boot-initial-env' ]";
+
+static struct xen_domain_cfg domu_cfg_2 = {
+	.name = "rpi_5_domu2",
```

Comment:
give it normal informative name - linux_pv_domu, or smth

Reply:
+